### PR TITLE
feat: use reduced keysize

### PR DESCRIPTION
**Pull request by dryajov**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,17 +12,17 @@
...
-    "release": "node --max-old-space-size=8192 $(npm bin)/aegir release ",
-    "release-minor": "node --max-old-space-size=8192 $(npm bin)/aegir release --type minor ",
-    "release-major": "node --max-old-space-size=8192 $(npm bin)/aegir release --type major ",
-    "coverage": "node --max-old-space-size=8192 $(npm bin)/aegir coverage --timeout 100000",
-    "coverage-publish": "anode --max-old-space-size=8192 $(npm bin)/aegir coverage --provider coveralls --timeout 100000"
+    "release": "node --max-old-space-size=8192 node_modules/.bin/aegir release ",
+    "release-minor": "node --max-old-space-size=8192 node_modules/.bin/aegir release --type minor ",
+    "release-major": "node --max-old-space-size=8192 node_modules/.bin/aegir release --type major ",
+    "coverage": "node --max-old-space-size=8192 node_modules/.bin/aegir coverage --timeout 100000",
+    "coverage-publish": "anode --max-old-space-size=8192 node_modules/.bin/aegir coverage --provider coveralls --timeout 100000"
```
Comment (diasdavid) [inline package.json]: @dryajov why this change?

Comment (dryajov) [reply package.json]: `$()` breaks windows as it is a unix shell (bash?) think.

Comment (diasdavid) [reply package.json]: Thanks!

File: package.json [hunk 2]
```diff
@@ -12,17 +12,17 @@
...
-    "release": "node --max-old-space-size=8192 $(npm bin)/aegir release ",
-    "release-minor": "node --max-old-space-size=8192 $(npm bin)/aegir release --type minor ",
-    "release-major": "node --max-old-space-size=8192 $(npm bin)/aegir release --type major ",
-    "coverage": "node --max-old-space-size=8192 $(npm bin)/aegir coverage --timeout 100000",
-    "coverage-publish": "anode --max-old-space-size=8192 $(npm bin)/aegir coverage --provider coveralls --timeout 100000"
+    "release": "cross-env NODE_OPTIONS='--max-old-space-size=8192' aegir release ",
+    "release-minor": "cross-env NODE_OPTIONS='--max-old-space-size=8192' aegir release --type minor ",
+    "release-major": "cross-env NODE_OPTIONS='--max-old-space-size=8192' aegir release --type major ",
+    "coverage": "cross-env NODE_OPTIONS='--max-old-space-size=8192' aegir coverage --timeout 100000",
+    "coverage-publish": "cross-env NODE_OPTIONS='--max-old-space-size=8192' aegir coverage --provider coveralls --timeout 100000"
```
Comment (diasdavid) [inline package.json]: 👍
